src/init.cpp

```diff
@@ -1777,6 +1777,7 @@ bool AppInitMain(InitInterfaces& interfaces)
     connOptions.nLocalServices = nLocalServices;
     connOptions.nMaxConnections = nMaxConnections;
     connOptions.nMaxOutbound = std::min(MAX_OUTBOUND_CONNECTIONS, connOptions.nMaxConnections);
```

Comment:
Nit: Perhaps this is a good opportunity to rename `nMaxOutbound` to something more precise... As it's easy to think that:

1. `nMaxOutbound` bounds `nOutbound` in `CConnman::ThreadOpenConnections(const std::vector<std::string>)`
1.  `nMaxOutbound` includes `nMaxOutboundBlocksOnly` added below

Both of which are untrue as far as I can tell.

Reply:
Bumping this. It seems the code changes would be slightly simpler if `nMaxOutbound` referred to the total of all outbound connections (including the blocksonly ones). That would also make it easier to reason whether no places where `nMaxOutbound` needed to be replaced with `nMaxOutbound + nMaxOutboundBlocksOnly` were forgotten.

Reply:
Your point appears to be underscored by the fact that I did overlook at least one place where nMaxOutbound needed to be replaced by the sum.  I'll try to improve this as you both suggest.

Reply:
Fixed now.